app/gateway-oss/2.5.x/getting-started/adding-consumers.md

```diff
@@ -11,8 +11,8 @@ title: Adding Consumers
   </ol>
 </div>
 
-In the last section, we learned how to add plugins to Kong, in this section
-we're going to learn how to add consumers to your Kong instances. Consumers are
+In the last section, we learnt how to add plugins to Kong, in this section
```

Comment:
```suggestion
In the last section, you learned how to add plugins to Kong, in this section
```

We favor US spelling in our docs set for consistency. I am also swapping "we" for "you" since you do that later on!